src/test/java/io/pravega/connectors/flink/FlinkPravegaOutputFormatTest.java

```diff
@@ -191,7 +191,7 @@ public String getRoutingKey(T event) {
                 .build();
 
         FlinkPravegaOutputFormat<String> spyFlinkPravegaOutputFormat = spy(flinkPravegaOutputFormat);
-        doReturn(clientFactory).when(spyFlinkPravegaOutputFormat).createClientFactory(stream.getScope(), clientConfig);
+        doReturn(clientFactory).when(spyFlinkPravegaOutputFormat).createClientFactory(anyString(), anyObject());
```

Comment:
Do we need this change for the build to pass or is this a stray change?

Reply:
The fix is for the build to pass